cpp/examples/lct_secir.cpp

```diff
@@ -47,25 +47,25 @@ int main()
                                                                 {50},  {10, 10, 5, 3, 2}, {20},         {10}};
 
     // Assert that initial_populations has the right shape.
-    if (initial_populations.size() != (int)LctState::InfectionState::Count) {
+    if ((int)initial_populations.size() != (int)LctState::InfectionState::Count) {
```

Comment:
```suggestion
    if (initial_populations.size() != (size_t)LctState::InfectionState::Count) {
```
Try to avoid narrowing if neither value is an int.